### PR TITLE
Cult of St. Credit, a new Chaplain sect and money sink. 

### PR DESCRIPTION
**Pull request by tarlockgod**
## About The Pull Request

I added a new sect for the chaplain, the Cult of St. Credit.
You can only pay the cult in holochips loaded with credits, everything else is heresy!
Joint today, become rich!

I also dont expect this to be merged(at least for now), its more of a first taste of byond coding for me. If the idea is liked though(besides the one guy in discord who told me to merge) i might expand it.

**EDIT: Besides some small fixes if any problems are found or pointed out, this PR is now feature complete**

## Why It's Good For The Game

Its gives the chap a bit more content, and a nice money sink for the assistants of the station, since the price is a money golem!
The golem rite might be a bit OP, but i think its fine for such a money sink.

## Changelog
:cl:
add: new sect for the chap to play with(also money sink, since you get favour with credits(holochips to be exact))
add: rite to summon some capitalist clothing
add: added bless mechanic, works the same as normal, but you pay 10 credits per hit. so run away or be forced to donate
/:cl:

### CODE REVIEW

**Comment by LemonInTheDark** [comment]
need a newline there 
image

Don't think this needs to be a draft, unless things are totally unfinished

**Comment by tarlockgod** [comment]
I made it a draft becasue i might expand it(Arcane also told me in my first PR to make it a draft instead).
Also, do you really think its good to merge?

**Comment by tarlockgod** [comment]
Ah good, newline fixed
If nobody complains i am gonna take it off draft tomorrow. Might add some new things if i get a good idea.

File: code/modules/religion/religion_sects.dm [hunk 1]
```diff
@@ -190,3 +190,23 @@
 	to_chat(L, "<span class='notice'>You offer [the_cell]'s power to [GLOB.deity], pleasing them.</span>")
 	qdel(I)
 	return TRUE
+
+/datum/religion_sect/capitalists
+	name = "The Cult of St. Credit"
+	desc = "A cult oriented around money."
+	convert_opener = "If you always donate your money and dont violate the NAP, you too might one day achieve the top 0.0000001%!<br>(Only Holochips accepted, for more questions reach out to our legal team!)"
+	alignment = ALIGNMENT_EVIL
+	desired_items = list(/obj/item/holochip)
```
Comment (Fikou) [inline code/modules/religion/religion_sects.dm:199]: why doesnt it accept space cash 😏

**Comment by Fikou** [comment]
you might want to multiply that cash needed for the golem rite by 10, its a really powerful thing

**Comment by ATH1909** [comment]
eh, I feel like this might ruin the "special"-ness of capitalist golems

make how much the bible effect heals someone dependent on how much cash they have in their ID (with a cap, of course, and maybe with a certain percent of that cash being automatically "donated" to the church on a successful heal)

**Comment by tarlockgod** [comment]
Good idea, i will have to see if i can do that

**Comment by tarlockgod** [comment]
Oh god economy is a bit too bigbrained for my right now, i am gonna try to finish it tomorrow.
But the chap going around extorting money for healing seems a bit OP unless i oull the numbers really down.

**Comment by optimumtact** [comment]
image

**Comment by tarlockgod** [comment]
I know oranges. Any thoughts on how to improve it? I can add more rites, or give them the bless-for-money thing that was suggested before. Its never gonna be merged anyway, so shoot.

**Comment by Firecage** [comment]
Honestly, the Golem thing is a bit to much. Considering that either Capitalist or Communist golems, in effect, makes something like Romerol look like the common cold compared to the Plague. They are, in effect, round enders of the worst kind due to both the conversion thing and the knock spell.

**Comment by tarlockgod** [comment]
That could indeed work, a, "church approved" version so to speak.
I am also gonna lower its cost then, since its not that extreme anymore.
With that and the sect bless i am working on, i would say the PR is in a good place.

**Comment by tarlockgod** [comment]
Since Finals are near and i dont wanna spend more time figuring out how to deduct money from peoples IDs upon being blessed, i am gonna let this PR leave draft. If it, somehow, gets merged i will add the bless after finals are done.

**Comment by Tlaltecuhtli** [comment]
.adjust_money(-value)  removes money, paste it in github to find similar places where its used so you know how the proc works,

and bless is triggered in /datum/religion_sect/proc/sect_bless(mob/living/L, mob/living/user)

**Comment by tarlockgod** [comment]
Thanks tlat, found the soulution to the payment problem thanks to your mention of the search function(didnt know it existed).

Also, this PR is now complete. Big thanks you to the people who commented here, you helped alot.

Even if it doesnt get merged, making it was a very informative journey into SS13 code.

**Comment by ExcessiveUseOfCobblestone** [comment]
blursed meme sect

**Comment by optimumtact** [comment]
You are responsible for this cobby

**Comment by Iamgoofball** [comment]
NUMBER
GO
UP

**Comment by ExcessiveUseOfCobblestone** [comment]
I dont want this in normal play as its a bit too memey for my tastes.

The options are TM for lols, configure it as admin-only, or close.

**Comment by tarlockgod** [comment]
Ye, its very memey. Admin only would fit for it. I could also downgrade the memefactor a bit since my finals are nearly done and i have time again.

**Comment by Iamgoofball** [comment]
Just ditch the golem bits and it'd be fine?

Scummy rich people cult is like 90% of religion, the fact we didn't have a cult for it already is absurd.

**Comment by tarlockgod** [comment]
Removed the golem, played around with some other rituals, didnt come up with anything good. If it doesnt get accepted i am gonna close it.

**Pull request closed without merging**